src/test/java/org/openrewrite/maven/AbstractRewriteMojoTest.java

```diff
@@ -18,7 +18,11 @@
 public class AbstractRewriteMojoTest {
 
     @ParameterizedTest
-    @ValueSource(strings = {"rewrite.yml", "https://httpstat.us/200"})
+    @ValueSource(strings = {
+        "rewrite.yml",
+        "https://httpstat.us/200",
+        "https://raw.githubusercontent.com/quarkusio/quarkus/main/jakarta/quarkus3.yml"
```

Comment:
Let's host the yaml file in the test case somewhere we control, rather than link to a quarkus github repo which might move at any time. Can check it in to rewrite-maven-plugin directly

Reply:
Sure, let me remove the 3rd test case since the "https://httpstat.us/200" covered this as well.